pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "jijmodeling-tutorial"
 version = "1.10.1"
 description = "Tutorial for JijModeling"
 readme = "README.md"
-requires-python = ">=3.9, <3.12" # jijmodeling does not support Python 3.12 yet
+requires-python = ">=3.9, <3.13" # jijmodeling does not support Python 3.13 yet
```

Comment:
https://github.com/Jij-Inc/JijModeling-Tutorials/blob/b17326397afaaf89dbdc6d7cca7a2a47ffb23ad6/.github/workflows/python.yml#L18 should also be upgraded

Reply:
jijmodeling-transpiler fails to build with 3.12 so we include upto 3.11.

Reply:
Elaborated in a4b4718.

Reply:
As a result, this pyproject.toml looks to require python version <3.12, and we should revert this line?

Reply:
Fixed in ae386ca5f0e113bb82d0553f5b8acc82770dfd37.